docker/php/Dockerfile

```diff
@@ -48,10 +64,7 @@ WORKDIR /srv/coopcycle
 COPY composer.json ./
 COPY composer.lock ./
 
-RUN mkdir -p \
-        var/cache \
-        var/logs \
-        var/sessions \
+RUN mkdir -p var/cache var/logs var/sessions \
```

Comment:
Remove trailing `\`

Reply:
Why? This will break the multi-line 🤔

Reply:
Forget it, I didn't look the whole file I thought it was the end of the line -_-